tests/feature/controllers/base.test.js

```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+
+const { app } = require('../../../src/app');
+
+
+describe('Test the root path', () => {
+    test('It should response the GET method', async () => {
+        const response = await supertest(app)
+            .get('/');
+
+        expect(response.statusCode).toBe(200);
```

Comment:
Consider using the "http-status-code" library that you've added to the business logic already, that was the main purpose actually :)